runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -11731,6 +11750,9 @@ executeBreakpointedBytecode: {
 #endif
 		, _objectAllocate(currentThread)
 		, _objectAccessBarrier(currentThread)
+if defined(J9VM_OPT_CRIU_SUPPORT)
```

Comment:
```suggestion
#if defined(DEBUG_VERSION) || defined(J9VM_OPT_CRIU_SUPPORT)
```
Similar comments here. Maybe we don't care about additional checking of `_dontSkipHooks` since it is debug mode anyway, otherwise, `#if !defined(DEBUG_VERSION)` can remove the references to `_dontSkipHooks`.